client/DOM.Map.js

```diff
@@ -296,18 +310,19 @@ define(['DOM.Common'], function (Common) {
             if (this.get('sharedEl') !== false) {
               this._add(atter(props), this._opt.rendered)
               var el = this._objectEls[props[atter.idIndex]]
-              el.style.visibility = 'hidden'
-              this.sc.transitions.updateUsing(null, options, this, function () {
-                el.style.visibility = ''
-              })
+              // el.style.visibility = 'hidden'
+              // this.sc.transitions.updateUsing(null, options, this, function () {
+              //   el.style.visibility = ''
+              // })
```

Comment:
Looks like it's better to remove it to keep the code clean

Reply:
It's a draft, it'll be fully rewritten. I am mostly done with this and hope to publish it in August if no more unexpected stuff pops up for me.